### PR TITLE
HostDashBoard: Update Dashboard Tabs

### PR DESCRIPTION
**Pull request by flickz**
Follows up https://github.com/opencollective/opencollective/issues/1975

This PR updates the dashboard tabs to `Expenses` --- `Donations` --- `Pending Applications` as shown below:

Screenshot 2019-10-10 at 12 41 46 PM

#### Todo
- [ ] Fix breaking test

### CODE REVIEW

**Comment by flickz** [comment]
I'm having doubt on the right icons to use for both Expenses & Donations, @Betree Can you suggest?

**Comment by Betree** [comment]
@flickz Since @alanna's first feedback on https://github.com/opencollective/opencollective/issues/1975 we've enforced the term `Financial Contributions`, let's use this term rather than `Donations`.

Regarding the icon we'll be able to change in the future, but I'd suggest the following from https://styled-icons.js.org/:

- Expenses: `styled-icons/material/Receipt`

image

- Financial Contributions `styled-icons/fa-solid/Donate`

image

File: components/host-dashboard/Dashboard.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```
Comment (Betree) [inline components/host-dashboard/Dashboard.js]: :eye_speech_bubble: Now that the `Expenses` and Orders part are clearly separated, maybe we should split in separate components?

File: pages/host.dashboard.js [hunk 2]
```diff
@@ -137,17 +137,29 @@ class HostDashboardPage extends React.Component {
...
+                <FormattedMessage id="host.dashboard.tab.donations" defaultMessage="Donations" />
               </MenuLink>
               <MenuLink
                 route="host.dashboard"
                 params={{ hostCollectiveSlug: slug, view: 'pending-applications' }}
                 isActive={view === 'pending-applications'}
               >
                 <CheckDouble size="1.2em" />
-                Pending applications
+                <FormattedMessage id="host.dashboard.tab.pendingApplications" defaultMessage="Pending applications" />
```
Comment (Betree) [inline pages/host.dashboard.js]: :+1: Nice that you've translated them all

**Comment by flickz** [comment]
@Betree I'm having difficulty resolving this conflict because of your recent changes, can you help so I don't mess it up?

**Comment by Betree** [comment]
Conflict is with https://github.com/opencollective/opencollective-frontend/commit/8baf6c45ed0604c641640dd2d9e0e7e1d6539161#diff-a374386f78ddb9c622c54ba7294017ec

From what I see the only conflicting line must be:

```es6
selectedCollectiveId={selectedCollective.id}
```

that became

```es6
defaultSelectedCollective={this.state.selectedCollective}
```

**Comment by znarf** [comment]
@flickz do you have an easy fix coming for the responsiveness or should we merge like that?

**Comment by flickz** [comment]
I think we should proceed to merge, I don't have a quick fix for the responsiveness. The dashboard responsiveness is an entire issue on it's own I plan to look into in coming days.

**Pull request merged**